modules/_labs/combobox/react/README.md

```diff
@@ -5,7 +5,12 @@
 </a>  This component is work in progress and currently in pre-release.
 
 A specialized input wrapper that adds an autocomplete list. Based on W3 spec for
-[List Autocomplete with Manual Selection](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html)
+[List Autocomplete with Manual Selection Aria 1.1](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html)
```

Comment:
Remove this line and the next. Maybe add a note as to why we're supporting the 1.0 spec and not the 1.1. I think it is fine to leave a link for the difference